server/routes/utils/dataReportHelpers.ts

```diff
@@ -145,8 +145,8 @@ export const getOpenSearchData = (
             if (typeof dateValue === 'string') {
               data._source[keys] = moment.utc(dateValue).tz(timezone).format(dateFormat);
             } else if (
-              fieldDateValue.length !== 0 &&
-              fieldDateValue instanceof Array
+              dateValue.length !== 0 &&
+              dateValue instanceof Array
             ) {
               fieldDateValue.forEach((element, index) => {
```

Comment:
Just for my knowledge, why whats the difference between `fieldDateValue ` and `dateValue ` above. Cuz I saw the change was trying to switch to `dateValue `.

Reply:
Originally the variable was `dateValue` for this condition but was mistakenly changed in different PR which caused `TypeError: Cannot create property '0' on number '1700813322772'`. Reverting to fix the issue.